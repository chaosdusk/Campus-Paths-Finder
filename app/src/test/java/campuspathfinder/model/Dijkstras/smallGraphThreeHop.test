CreateGraph graph

AddNode graph node1
AddNode graph node2
AddNode graph node3
AddNode graph node4
AddNode graph node5
AddNode graph node6

AddEdge graph node1 node2 4.0
AddEdge graph node1 node3 5.0
AddEdge graph node2 node4 6.0
AddEdge graph node3 node5 4.0
AddEdge graph node4 node6 4.0
AddEdge graph node5 node6 4.0

FindPath graph node1 node6