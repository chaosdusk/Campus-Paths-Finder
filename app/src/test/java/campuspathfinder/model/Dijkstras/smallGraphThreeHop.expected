created graph graph

added node node1 to graph
added node node2 to graph
added node node3 to graph
added node node4 to graph
added node node5 to graph
added node node6 to graph

added edge 4.000 from node1 to node2 in graph
added edge 5.000 from node1 to node3 in graph
added edge 6.000 from node2 to node4 in graph
added edge 4.000 from node3 to node5 in graph
added edge 4.000 from node4 to node6 in graph
added edge 4.000 from node5 to node6 in graph

path from node1 to node6:
node1 to node3 with weight 5.000
node3 to node5 with weight 4.000
node5 to node6 with weight 4.000
total cost: 13.000